tests/test_live.py

```diff
@@ -0,0 +1,265 @@
+import io
+import os
+import random
+import sys
+import time
+from typing import List
+
+import pytest
+from rich.console import Console
+from rich.live import Live
+from rich.table import Table
+from rich.text import Text
+
+from tests.util import get_capture_text, set_capture_text
+
+
+def create_capture_console(*, width: int = 60, height: int = 80) -> Console:
+    return Console(
+        width=width,
+        height=height,
+        file=io.StringIO(),
+        force_terminal=True,
+        legacy_windows=False,
+        color_system=None,  # use no color system to reduce complexity of output
+    )
+
+
+def create_base_table() -> Table:
+    table = Table(title="test table", caption="table caption", expand=True)
+    table.add_column("foo", footer=Text("total"), no_wrap=True, overflow="ellipsis")
+    table.add_column("bar", justify="center")
+    table.add_column("baz", justify="right")
+
+    return table
+
+
+def check_output(output_file: str, output: str) -> None:
+    output = output.replace("\r", "")
+    if os.getenv("CAPTURE") is not None:  # adjust the correct output check
+        set_capture_text("live", output_file, output=output)
+
+    correct_output = get_capture_text("live", output_file).replace("\r", "")
+
+    if os.getenv("VERBOSE") is not None:
+        print("Current Output\n", output)
+        print("Correct Output\n", correct_output)
+
+    assert output == correct_output, "Console output differs from the correct output"
+
+
+def test_live_state() -> None:
+
+    with Live("") as live:
+        assert live.is_started
+        live.start()
+
+        assert live.item == ""
+
+        assert live.is_started
+        live.stop()
+        assert not live.is_started
+
+    assert not live.is_started
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7 or higher")
```

Comment:
What's the reason for skipping 3.6? Rich supports 3.6.1 or above...

Reply:
The issue is I could not get the tests to pass on 3.6, I believe it is how python 3.6 handles utf-8 encoding. Perhaps with simpler tests it may not be an issue.

Reply:
I'm not aware of any differences in utf-8 encoding. You might want to investigate what's different there.

Reply:
Ok, I will reinvestigate once finished the other changes and making the tests use simpler renderables.